### PR TITLE
Removing remaining .blend file (for the desert)

### LINKED ISSUE #276: Unity shows three "Blender could not be found." errors (even though Blender is no longer required)
in PR #212 (issue #178), I removed the Blender dependency (i.e., `.blend` files are no longer used directly from within the Unity project). for some reason, upon a fresh `git clone` of this repo and opening the Unity project folder, I see these three errors for `Blender could not be found.`:

screenshot of Unity's "Blender could not be found." errors

I haven't dug around in a bit, but it could be because there are Blender source files (`.blend`) in the project directory. I'm not entirely sure.

### PR DESCRIPTION
**Pull request by delapuente**
Fix #276

### CODE REVIEW

**Comment by delapuente** [comment]
The solution @cvan describes in #276 says he removed the `.blend` files for the hands. If we want to keep them, we should move all the sources out of the Unity folder so Unity would not complain about them.

**Comment by caseyyee** [comment]
@delapuente silly idea that may work: maybe try to zip them up?

**Comment by cvan** [comment]
can we just move them into a separate `art` or `branding` directory at the top level? this way the [_Unity Asset Store Tools_](https://assetstore.unity.com/packages/KATEGORY-MISSING/asset-store-tools-115) would not see any `.blend` files in the [`Assets` directory](https://github.com/mozilla/unity-webvr-export/tree/master/Assets) that gets inspected when uploading the package assets.

thoughts, y'all?

**Comment by caseyyee** [comment]
That seems like a reasonable solution as well.  maybe just create a `/art-assets` (or something similarly named) folder.

**Comment by delapuente** [comment]
Agree, then it makes sense to recover the Blender files for the hands and move into the same folder. I named it `ArtSources`.

**Comment by delapuente** [comment]
Done. Is it OK?

File: Assets/Models/Desert Environment/desert-environment.blend.meta [hunk 1]
```diff
@@ -1,117 +0,0 @@
-fileFormatVersion: 2
```
Comment (cvan) [inline Assets/Models/Desert Environment/desert-environment.blend.meta]: is this file not needed anymore?

**Comment by delapuente** [comment]
No, that does not make sense, Metafiles are just for Unity assets which should be inside assets. This is outside the asset folder so it is useless to move the metafile too.

**Comment by cvan** [comment]
> No, that does not make sense, Metafiles are just for Unity assets which should be inside assets. This is outside the asset folder so it is useless to move the metafile too.

if the Blender sources aren't used as Unity assets anymore, would it make sense to remove them?

I'm approving and merging the changes. if you have thoughts about this, let me know - I can make the changes, or feel free to open a follow-up PR/issue, and I'll review it ASAP.

thanks, @delapuente!

**Comment by delapuente** [comment]
That was the solution in the first place but, per [@caseyyee comment](https://github.com/mozilla/unity-webvr-export/pull/279#pullrequestreview-119670262), we opted to include the source files in a separate folder.

**Pull request merged**